src/build/vite/plugin.ts

```diff
@@ -14,6 +13,7 @@ import * as rou3 from "rou3";
 import * as rou3Compiler from "rou3/compiler";
 import { resolveModulePath } from "exsolve";
 import { prettyPath } from "../../utils/fs";
+import path from "node:path";
```

Comment:
We could use pathe (already a dep i guess vite also uses same) to make sure paths are always normalized